app/src/main/java/com/garethevans/church/opensongtablet/performance/PerformanceFragment.java

```diff
@@ -800,14 +800,28 @@ public void onGlobalLayout() {
 
                 // Pass this scale factor to the zoom layout as the new minimum scale
                 myView.zoomLayout.setCurrentScale(scaleFactor);
-
                 myView.zoomLayout.setSongSize(widthAfterScale,heightAfterScale);
 
-                myView.zoomLayout.post(() -> {
+                // Set the load status to the song (used to enable nearby section change listener)
+                mainActivityInterface.getSong().setCurrentlyLoading(false);
+
+                // Release the processing lock
+                processingTestView = false;
+
+                // Set the positions
+                mainActivityInterface.getDisplayPrevNext().getPositions();
+
+                // Slide in
+                long QOSAdjustment = doSongLoadQOSTime - (System.currentTimeMillis() - doSongLoadStartTime);
```

Comment:
What is 'doSongLoadQOSTime'
This doesn't appear anywhere in the pull request

Reply:
I have a commit that addresses making the app more Material.  I suspect bits have crept into this commit - sorry.
As I did not want to overwhelm you I deferred commit which is now hampering my ability to work on any other fixes.  This is a 'make it material' commit - with more material standard layout and icon use.  A similar thing happened to v5.

This is improving the beta, to get it looking good.

I can prepare a pull?

Enh: Action bar
- Fixes for variable clock size, button positions and glitches when moving around the option menu
 - Item sizes and positioning made consistent
 - Option menu transitions made cleaner.
 - Help icon moved into option menu
 - Fragment calls of updateToolBarHelp moved to onResume to ensure help button onclick action
 - Title and Battery info click is active only on top option menu
 - Battery info is 'gone' when not in use
 - Whole title area now clickable.

Enh: 'Material' styling applied.
- Flatter dialog tops
- 'settings_outline', 'help_outline' and 'web_outline' 24dp icons from google used in relevant places in app. Other navigation icons set to 24dp.  The app matched with the material styling of storage, cast and other system dialogs.

Fix: Custom margins
- Are re-applied on orientation change
- Song menu position now respects custom margins.
- An open Song menu is closed on orientation change

Enh: 30% faster song transitions
- v5 optimisations appled, song animate out is short and combines with a QOS behavior to give a user perception of consistent speed (now faster) of song change.